### ISSUE TITLE
bug(windows): The Khmer Angkor keyboard in LibreOffice 7.6  is not giving the expected outcome in the Writer document

### ISSUE THREAD

**Issue by bharanidharanj** [keymanapp/keyman#9840]
### Describe the bug

 After switching to the Khmer Angkor keyboard, I observed that when I entered "xEjmr" in a LibreOffice Writer document, it did not produce the desired outcome. 'ខ្មែរ'. 

### Reproduce the bug

1. Install Keyman 17.0.196-alpha build.
2. Download and Install Khmer Angkor Keyboard. 
3. Upgrade LibreOffice to 7.6 version. 
4. Open LibreOffice Writer document. 
5. Switch to Khmer Angkor keyboard. 
6. Type 'xEjmr'.
Here, I noticed that it is showing  'ខរ ' words instead of 'ខ្មែរ'. Seems to be an issue. 

Note: It is working fine when I use the Notepad App. 

I have attached the Screenshot for reference. 

..working fine in Notepad App

   <img width="200" src="https://github.com/keymanapp/keyman/assets/19683143/b2bf0d63-fc75-42c3-b235-ecdf633a7e0d">

..Not working in LibreOffice Writer 7.6

   <img width="300" src="https://github.com/keymanapp/keyman/assets/19683143/b3ff56f1-e9ec-45a9-b244-d6dd9d8196fa">

### Expected behavior

It should output 'ខ្មែរ' when I typed 'xEjmr' in the LibreOffice Writer document. 

### Keyman apps

- [ ] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [ ] Keyman for macOS
- [X] Keyman for Windows
- [ ] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

17.0.196-alpha

### Operating system

Windows 10 OS, Windows 11 OS (VM)

### Keyboard name

khmer_angkor

### Keyboard version

1.3

### Language name

Khmer

**Comment by rc-swag** [comment]
Investing this bug has shown how "helpful" programs like Word and LibreOffice can sometimes make it hard to know what is going on. 
I installed version 7.6 of LibreOffice and followed the steps above and it was working as expected for me. Then I noticed LibreOffice was automatically changing the font to `Khmer UI`. The screen shot attached to this PR I can see the font is "Arial"
I changed the font to Arial and typed the sequence I got the same incorrect visual result as above. However if I change the font back to a Khmer font it displays directly. If I chose Arial again type the sequence (it renders incorrectly) I then copy the text and paste it into the Keyman Developer test webpage it shows that it is indeed the correct code points. It also renders correctly.

u1781 + u17d2 + u1798 + u17c2 + u179a

If I paste into MS Word it automatically changes the font to a font that supports Khmer. It will not let me change it back to Arial at all!!!.  If type Notepad with Arial it works fine.

@bharanidharanj  Can you please run your test again in LibreOffice but this time choose a Khmer font.

**Comment by bharanidharanj** [comment]
> Investing this bug has shown how "helpful" programs like Word and LibreOffice can sometimes make it hard to know what is going on. I installed version 7.6 of LibreOffice and followed the steps above and it was working as expected for me. Then I noticed LibreOffice was automatically changing the font to `Khmer UI`. The screen shot attached to this PR I can see the font is "Arial" I changed the font to Arial and typed the sequence I got the same incorrect visual result as above. However if I change the font back to a Khmer font it displays directly. If I chose Arial again type the sequence (it renders incorrectly) I then copy the text and paste it into the Keyman Developer test webpage it shows that it is indeed the correct code points. It also renders correctly.
> 
> u1781 + u17d2 + u1798 + u17c2 + u179a
> 
> If I paste into MS Word it automatically changes the font to a font that supports Khmer. It will not let me change it back to Arial at all!!!. If type Notepad with Arial it works fine.
> 
> @bharanidharanj Can you please run your test again in LibreOffice but this time choose a Khmer font.

@rc-swag Sure, I will try that.

**Comment by bharanidharanj** [comment]
@rc-swag I retested as per your suggestion and here is my observation: 1. In the LibreOffice writer document, I changed the font to 'Khmer Mondulkiri' , then I tried the same word 'xEjmr'. This time, it shows the expected output on the document. 2. Seems to be working fine after changing the Font in the Writer document. Thanks for the suggestion, Ross. I think it would be better to close this issue. Thanks. 

   <img width="300" src="https://github.com/keymanapp/keyman/assets/19683143/ebfb7adb-321f-4928-a84a-097f0fd0b214">
